pegasus/sites.v3/code.org/views/about_headshots.haml

```diff
@@ -1,24 +1,23 @@
-#headshots{:style=>'width: 790px; padding: 0; margin: 0;'}
-  - people.each_slice(columns) do |group|
-    .headshot_row{:style=>'width: 790px; padding: 0; margin: 0;'}
-      - group.each do |person|
-        - column_width = columns==3 ? '260' : '160'
-        .headshot_column{:style=>'width: ' + column_width + 'px; padding: 15px; margin: 0; float: left;'}
-          - avatar_url = avatar_image(person[:name_s], 200, true)
-          - unless person[:url_s].nil_or_empty?
-            %a{:href=>person[:url_s], :target=>"_self"}
-              .avatar_container{:style=>'height: 200px;'}
-                - unless avatar_url.nil_or_empty?
-                  %img{:src=>avatar_url, :style=>'max-height: 200px; border-radius: 100px;'}
-              %h3{:style=>'height:20px'}
-                = person[:name_s]
-          - else
-            .avatar_container{:style=>'height: 200px;'}
-              - unless avatar_url.nil_or_empty?
-                %img{:src=>avatar_url, :style=>'max-height: 200px; border-radius: 100px;'}
-            %h3{:style=>'height: 20px;'}
+
+#headshots{:style => 'margin-top: 20px;'}
+- people.each_slice(3) do |trio|
+  .headshot_row{:style=>'width: 100%; padding: 0; margin: 0;'}
+    - trio.each do |person|
+      - avatar_url = avatar_image(person[:name_s], 200, true)
+      .col-33.avatar_container{ :id => person[:id] }
```

Comment:
I don't think we have an official rule on this style, but just to keep it consistent with the rest of the file can you remove the spaces? And can you do the same style cleanup for `about_logos` and `target_languages`?

`.col-33.avatar_container{:id=>person[:id]}`